recipes/codecov/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "codecov" %}
+{% set version = "2.0.6" %}
+{% set sha256 = "999bf4d6f7e072433601798f3f291277a4ab519b245af2971cdeb247dbb21454" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+#  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/codecov/codecov-python/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - coverage
+    - requests >=2.7.9
+    - argparse  # [py26]
+  run:
+    - python
+    - coverage
+    - requests >=2.7.9
+    - argparse  # [py26]
+
+test:
+  requires:
+    - coverage
+    - ddt
+    - mock
+    - pytest
+    - pytest-cov
+    - funcsigs
+    - requests
+    - unittest2
+  source_files:
+    - tests/*
+  commands:
+    - py.test
```

Comment:
I usually avoid running unit tests in the conda packaging process and rely on the upstream package to run its unit tests. That being said, if you would prefer to run tests then just respond here indicating that you would prefer to do that and I'll help you make the necessary changes to get these tests running. 

In any event, you definitely want to make sure that you can import codecov and that the codecov CLI is working. To that end, you at least want this as your test section:

```yaml
test:
  imports:
    - codecov
  commands:
    - codecov -h
```

FWIW, that's *all* that I'd include in the test section